### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by rst77**
Signed-off-by: Renato Teixeira <renato@renatoteixeira.com>

A project to translate the Fabric documentation into Brazilian portuguese.

### CODE REVIEW

File: pt_BR/docs/Pipfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+[dev-packages]
+
+[packages]
+Sphinx = "==1.7.2"
+sphinx-rtd-theme = "*"
+recommonmark = "==0.4.0"
+
+[requires]
+python_version = "3.7"
```
Comment (pamandrejko) [inline pt_BR/docs/Pipfile:14]: Renato - do you want to update your Pipfile to be the shorter version?
See https://github.com/hyperledger/fabric/pull/1394/files
```
[packages]
Sphinx = "==1.8.5"
recommonmark = "==0.4.0"
```

**Review by pamandrejko — COMMENTED** [review]
Great to see this being populated Renato!

File: pt_BR/docs/Pipfile (hunk 1 above)
Comment (rst77) [reply pt_BR/docs/Pipfile]: It make sense, I did the adjustments in both Pipenv files

**Comment by btl5037** [comment]
Renato, I will try to get the CI set up for this ASAP, just a little swamped at the moment. But I will try to prioritize this for tomorrow

**Comment by ODOWDAIBM** [comment]
@rst77 delighted to see this Renato -- thank you. @aneenaalexander Have a look at this structure -- as discussed, I'd like very much to bring the ML repo into this structure.

**Comment by aneenaalexander** [comment]
@rst77 happy to see this Renato-- @ODOWDAIBM Sure Anthony!! WIll reorganize fabric-docs-ml structure complying to  new i18n structure

**Pull request merged**